### PR TITLE
iOS: Fetch sessions as model

### PR DESCRIPTION
**Pull request by takahirom**
## Overview (Required)
- Now, we can use session data.
- We can open and confirm it by opening frontend/ios with Xcode.

## Screenshot
image

### CODE REVIEW

**Comment by takahirom** [comment]
Let me merge for calling to iOS engineers

**Comment by takahirom** [comment]
If you can comment here, we will give you a PR for improvement separately!

File: data/api-impl/src/iosMain/kotlin/io/github/droidkaigi/confsched2019/data/api/MainLoopDispatcher.kt [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+import kotlinx.coroutines.Runnable
+import platform.darwin.DISPATCH_TIME_NOW
+import platform.darwin.dispatch_after
+import platform.darwin.dispatch_async
+import platform.darwin.dispatch_get_main_queue
+import platform.darwin.dispatch_time
+import kotlin.coroutines.CoroutineContext
+
+@UseExperimental(InternalCoroutinesApi::class)
+object MainLoopDispatcher : CoroutineDispatcher(), Delay {
```
Comment (kikuchy) [inline data/api-impl/src/iosMain/kotlin/io/github/droidkaigi/confsched2019/data/api/MainLoopDispatcher.kt]: Awesome!!!!!!!! 🎉✨㊗️💪🔥

File: buildSrc/src/main/java/dependencies/Dep.kt [hunk 2]
```diff
@@ -107,11 +107,13 @@ object Dep {
     }
 
     object Ktor {
-        val version = "1.0.1"
+        val version = "1.1.1"
         val clientCommon = "io.ktor:ktor-client-core:$version"
         val clientAndroid = "io.ktor:ktor-client-okhttp:$version"
+        val common = "io.ktor:ktor-client-json:$version"
```
Comment (kikuchy) [inline buildSrc/src/main/java/dependencies/Dep.kt]: `commonJson` is more better I think.

Comment (takahirom) [reply buildSrc/src/main/java/dependencies/Dep.kt]: Probably we should use `jsonCommon` 👍

**Pull request merged**